src/components/PriceField/PriceField.tsx

```diff
@@ -101,7 +129,7 @@ export const PriceField: React.FC<PriceFieldProps> = props => {
         ),
         inputProps: {
           min: 0,
-          step: "0.01",
+          step: 1 / Math.pow(10, maxDecimalLength),
```

Comment:
```suggestion
          step: 1 / 10 ** maxDecimalLength,
```
there isn't any implementation differences between these, but imo we should use es7 syntax, as it's cleaner